### DISCUSSION TITLE
I'm not able to update the status of task

### DISCUSSION THREAD

**Discussion by vidhi-pavan-agrawal** [Netflix/conductor#2506] [General]
I'm using Netflix Conductor v3.3.1 I have setup conductor using docker-compose while trying out kitchensink example I'm able to poll the tasks and would see  the task queue using :
curl http://localhost:8080/api/tasks/poll/task_1
to update the task_1 status as complete and push task5 in the queue I used with the workflowInstanceID and TaskID received from earlier command:
curl -H 'Content-Type:application/json' -H 'Accept:application/json' -X POST http://localhost:8080/api/tasks/ -d '
{
    "taskId": "b9eea7dd-3fbd-46b9-a9ff-b00279459476",
    "workflowInstanceId": "b0d1a935-3d74-46fd-92b2-0ca1e388659f",
    "status": "COMPLETED",
    "outputData": {
        "mod": 5,
        "taskToExecute": "task_1",
        "oddEven": 0,
        "dynamicTasks": [
            {
                "name": "task_1",
                "taskReferenceName": "task_1_1",
                "type": "SIMPLE"
            },
            {
                "name": "sub_workflow_4",
                "taskReferenceName": "wf_dyn",
                "type": "SUB_WORKFLOW",
                "subWorkflowParam": {
                    "name": "sub_flow_1"
                }
            }
        ],
        "inputs": {
            "task_1_1": {},
            "wf_dyn": {}
        }
    }
}'
this is just returning the old task id and was not able to update the status of the tasks as "COMPLETE" .

**Comment by jxu-nflx** [comment]
Hello @vidhi-pavan-agrawal, if I understand correctly, you are trying to update task_1 as completed via POST to http://localhost:8080/api/tasks/ using the taskId and workflowInstanceId of task_1, but it didn't work. Could you try a GET of that task to include more details for us to take a look at? Also, after you poll task_1 from the queue, did you see the status of task_1 got updated to "in progress"?

**Reply by vidhi-pavan-agrawal** [reply]
hello , I could see the task status as completed through http://localhost:8080/api/tasks/{task_ID} but the same is not reflected on the conductor  UI  http://localhost:5000/ (all the settings are kept as default)
![Screenshot (192)](https://user-images.githubusercontent.com/73054373/136691933-a542d600-15a9-4b12-8194-41640964b04c.png)

**Comment by jxu-nflx** [comment]
Hello @vidhi-pavan-agrawal I think you are looking at the wrong place, the screenshot above shows the workflow definition. If you want to see workflow execution details, try use the "Execution" tab to find it

**Reply by vidhi-pavan-agrawal** [reply]
well I had checked on the "Execution" tab as well there also can't see any details, as mentioned in the documentation where we can clearly see which tasks are scheduled( in yellow) and tasks completed (in green ) I'm not able to see the same in my UI even after pushing proper commands  https://netflix.github.io/conductor/labs/beginner/

**Reply by jxu-nflx** [reply]
Could you add a snapshot of workflow execution? You should be able to see execution details like this
![Screen Shot 2021-10-11 at 2 04 53 PM](https://user-images.githubusercontent.com/90653611/136855480-c4ee0fd1-5b51-48c9-82ed-8ff4cbc69f20.png)

**Reply by jxu-nflx** [reply]
The reason you couldn't see status might be Elasticsearch is not configured to run locally. Could you please check the Elasticsearch status, and that workflow and task data are being written successfully to Elasticsearch to make sure you can retrieve them via UI successfully

**Reply by vidhi-pavan-agrawal** [reply]
I have tried the installation via docker-compose . So now how do I configure Elasticsearch to run locally? any suggestions will be helpful.

**Reply by vidhi-pavan-agrawal** [reply]
I just want to try some simple workflow examples with Conductor, for demo purpose

**Reply by spacioustime** [reply]
I'm running into the same issue. I changed conductor.indexing.enabled to true in /server/config/config-local.properties based on a google search that led me to https://github.com/Netflix/conductor/issues/2175. After I did that the UI stopped showing the definitions too.

**Reply by arvin-meghraj** [reply]
I'm encountering the same issue. 

I've checked the config file and indexing is indeed enabled (`conductor.indexing.enabled=true`). UI works for definitions though.

I can see that the Task Queue Size is being updated in the UI when I poll for a task. GET on taskId returns the task details with status as `COMPLETED`.

`http://localhost:8080/api/workflow/running/{workflow-name}` also returns the list of executions triggered.

**Comment by jxu-nflx** [comment]
Other than setting `conductor.indexing.enabled=true` , also need to add `conductor.elasticsearch.url=http://es:9200` since the dock compose uses [internal network model](https://github.com/Netflix/conductor/blob/main/docker/docker-compose.yaml#L70). By default, that url is setting to [localhost](https://github.com/Netflix/conductor/blob/main/server/src/main/resources/application.properties#L44) which will fail in this case.

**Reply by arvin-meghraj** [reply]
I have property set to http://es:9200 as well.

**Reply by jxu-nflx** [reply]
Weird, I had that running fine locally. Try login to your running container of conductor-server via command: `docker exec -it ${{your_conductor_server_container}} bash` and check the configs within file `/app/config/config-local.properties` to make sure it got the right configs

**Reply by jxu-nflx** [reply]
A PR created to enable that by default: https://github.com/Netflix/conductor/pull/2519
